msbuild/Xamarin.Mac.Tasks.Core/Tasks/MmpTaskBase.cs

```diff
@@ -209,6 +210,8 @@ protected override string GenerateCommandLineCommands ()
 				
 			if (IsAppExtension)
 				args.AddQuotedLine ("/extension");
+			if (IsXPCService)
```

Comment:
No supper important, but we are mixing methods with the PR that use XPC with others in the bindings that use Xpc. I could choose one and be consistent.